.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Helm Charts
+
+on:
+  push:
+    tags: ['*']
+
+env:
+  helm-version: v3.8.1
+
+jobs:
+  publish-chart:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
Nit, not sure if needed? as we already filter on tags on line 5?
```suggestion
```

Reply:
I would like to keep it as a safety check if we ever change line 5